src/field.rs

```diff
@@ -817,7 +817,7 @@ impl Integer for u128 {
 make_field!(
     /// Same as Field32, but encoded in little endian for compatibility with Prio v2.
```

Comment:
FWIW this comment is out of date (had me confused for a bit). We changed all field encodings to be little-endian in #495, following a change in draft-irtf-cfrg-vdaf-05. Thus, Prio v2 and VDAF all use the same encoding.